src/products/firefox_desktop/review.md

```diff
@@ -4,140 +4,98 @@
 
-The localizations for all channels can be found in [l10n-central](https://hg.mozilla.org/l10n-central/), with a single repository for each locale.
+Localized strings for all channels can be found in [firefox-l10n](https://github.com/mozilla-l10n/firefox-l10n), with a single subfolder for each locale.
 
 The review process consists of three parts:
-* Review strings landing in `mozilla-central`. Currently `comm-central` doesn’t undergo a similar review process.
-* Review strings landing in `gecko-strings-quarantine`. The quarantine repository is updated twice a day via automation in [Task Cluster](https://firefox-source-docs.mozilla.org/taskcluster/kinds.html#l10n-cross-channel).
-* Push reviewed strings to `gecko-strings`, and start the localization process.
+* Review strings landing in `mozilla-central`.
+* Review strings extracted into the `update` branch of `firefox-l10n-source`. The [automation](https://github.com/mozilla-l10n/firefox-l10n-source/blob/main/.github/workflows/update.yml) runs twice a day in the repository with any changes added to a pull request against the `main` repository.
+* Review and merge strings in the pull request to `main`, and start the localization process.
+
+Things to look out for during the review process:
+* Unclear strings and missing localization comments: the best way to identify them is to translate the strings, only having the string and comment as context (not the entire file, or the bug). For example: is the word used as both a noun and a verb in English? Is the ID clear enough to give context (e.g. `buttonLabel`)?
+* String changes without new IDs.
+* Duplicated strings.
+* [Localization issues](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html), like misused plural forms, unclear comments, etc.
 
 ### Review strings landing in mozilla-central
 
 You can get the list of changesets touching localized strings in the last 2 days from [mozilla-central](https://hg.mozilla.org/mozilla-central/log?rev=keyword("locales/en-US")+and+pushdate("-2")). Adjust the `pushdate` part if you want to see more or less days.
 
+Note: this review includes changes landing in `mozilla-central`, but those changes may not have landed yet. Once changes have completed the landing process, those strings will also be available to review in the next step - [Review strings in update branch of firefox-l10n-source](#review-strings-in-update-branch-of-firefox-l10n-source).
+
 There are some unrelated changesets, like en-US dictionary updates, but the majority of landings are relevant and need to be checked for localization issues.
```

Comment:
We should call out "preview" strings at this point, since they're more and more common.